projects/valtimo/zgw/src/lib/modules/documenten-api/components/documenten-api-tag-modal/documenten-api-tag-modal.component.html

```diff
@@ -0,0 +1,58 @@
+<!--
+  ~ Copyright 2015-2024 Ritense BV, the Netherlands.
+  ~
+  ~ Licensed under EUPL, Version 1.2 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<cds-modal
+  valtimoCdsModal
+  *ngIf="{
+    disabled: disabled$ | async
+  } as obs"
+  [open]="open"
+  size="sm"
+>
+  <cds-modal-header showCloseButton="true" (closeSelect)="onClose()">
+    <h3 cdsModalHeaderHeading>
+      {{ 'zgw.tags.addModal.header' | translate }}
+    </h3>
+  </cds-modal-header>
+
+  <section cdsModalContent>
+    <ng-container *ngTemplateOutlet="tagForm; context: {obs: obs}"></ng-container>
+  </section>
+
+  <cds-modal-footer>
+    <ng-container *ngTemplateOutlet="buttons; context: {obs: obs}"></ng-container>
+  </cds-modal-footer>
+</cds-modal>
+
+<ng-template #tagForm let-obs="obs">
+  <form [formGroup]="tagFormGroup" class="tag-form">
+    <cds-label>
+      {{ 'zgw.tags.fields.value' | translate }}
+
+      <input cdsText formControlName="value" [attr.modal-primary-focus]="true" />
+    </cds-label>
+  </form>
+</ng-template>
+
+<ng-template #buttons let-obs="obs">
+  <button cdsButton="secondary" (click)="onClose()" [disabled]="obs.disabled">
+    {{ 'interface.cancel' | translate }}
+  </button>
+
+  <button cdsButton="primary" [disabled]="invalid || pristine || obs.disabled" (click)="addTag()">
+    {{ 'interface.add' | translate }}
+  </button>
+</ng-template>
```

Comment:
In this case I don't think content projection is necessary. If you put it directly in the section and footer you can also omit sending the context